common/changes/@microsoft/api-extractor/dont-trim-imports_2024-02-21-00-32.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "fix: Don't trim imports from external packages based when generating rollups",
```

```suggestion
      "comment": "Fix an issue where imports were trimmed from external packages based when generating .d.ts rollups",
```
https://rushjs.io/pages/best_practices/change_logs/